search/search.md

```diff
@@ -127,6 +127,13 @@ tag=netflow netflow Src Dst
 
 Further examples throughout the documentation should help clarify the use of enumerated values.
 
+(intrinsic_enumerated_values)=
+## Intrinsic Enumerated Values
+
+Intrinsic Enumerated Values are Enumerated Values which are created at the time of ingest. Intrinsic enumerated values are optionally created by ingesters and often contain metadata or pre-processed extractions. Use the [intrinsic](intrinsic/intrinsic) module to attach and work with intrinsic enumerated values.
```

Comment:
Again, the use of "attach" here is confusing, ESPECIALLY since we talk about an Attach config stanza in ingesters.

The attaching happens at ingest. We need a better word for what happens at search time.